book/docs/setup/editors/vscode.md

```diff
@@ -17,4 +18,60 @@ _This guide assumes that you've completed the steps described in [Prerequisites]
    ```
    _Note: if you're using `nix-shell`, make sure to run it first._
 
-If everything is fine you should get auto-completion and other features working 🎉
\ No newline at end of file
+If everything is fine you should get auto-completion and other features working 🎉
+
+---
+</br>
+
+### Additional Troubleshooting Steps
+_In the event the above steps result in the Visual Studio Code Editor not displaying definitions._
+
+### Tested with
+```
+Ubuntu                                          v20.x
+Windows                                         v10.x
+Nix                                             v2.3.10
+Visual Studio Code                              v1.55.2
+VSCode Extension Haskell                        v1.2.0
+VSCode Extension Haskell Syntax Highlighting    v3.4.0
+```
+
+### How to Check
+_Select a function in the Visual Studio Code Editor and right click and choose **Go to Definition** or highlight a function and press **F12**_.
+
+`Example: bindMaybe displays the error No definition found for 'bindMabye' even though it is defined only a few lines up.`
+```
+bindMaybe :: Maybe a -> (a -> Maybe b) -> Maybe b
+bindMaybe Nothing _ = Nothing
+bindMaybe (Just x) f = f x
+
+foo :: String -> String -> String -> Maybe Int
+foo x y z =  readMaybe x `bindMaybe` \num1 ->
+             readMaybe y `bindMaybe` \num2 ->
+             readMaybe z `bindMaybe` \num3 ->
+             Just (num1 + num2 + num3)
+```
+
+### Possible Solution
+1. Open the built-in terminal inside your Visual Studio Code by pressing `control` _+_ `~ ` at the same time or by going to `Terminal > New Terminal` within the file menu.
+
+2. From the built in terminal, navigate to your **plutus** directory or clone a new copy of the [plutus repository](https://github.com/input-output-hk/plutus).
```

Comment:
```suggestion
2. From the built-in terminal, navigate to your **plutus** directory or clone a new copy of the [plutus repository](https://github.com/input-output-hk/plutus).
```

Keeping consistent with 1.

Reply:
I totally agree! Resolved in commit c1a467a on `getHashSet/docs.plutus-community.com` branch `editors`.